test/puppetlabs/puppetdb/pdb_routing_test.clj

```diff
@@ -0,0 +1,89 @@
+(ns puppetlabs.puppetdb.pdb-routing-test
+  (:require [clojure.test :refer :all]
+            [clj-http.client :as client]
+            [puppetlabs.puppetdb.testutils.services :as svc-utils]
+            [puppetlabs.puppetdb.testutils :as tu]
+            [puppetlabs.puppetdb.cheshire :as json]
+            [puppetlabs.puppetdb.time :as time]
+            [puppetlabs.puppetdb.client :as pdb-client]
+            [clj-time.coerce :refer [to-string]]
+            [clj-time.core :refer [now]]
+            [puppetlabs.puppetdb.testutils.dashboard :as dtu]
+            [puppetlabs.puppetdb.utils :as utils]
+            [puppetlabs.puppetdb.cli.services :as clisvc]
+            [puppetlabs.puppetdb.pdb-routing :refer :all]
+            [puppetlabs.trapperkeeper.app :as tk-app]))
+
+(defn command-base-url [{:keys [host port]}]
+  (utils/pdb-cmd-base-url host port :v1))
```

Comment:
Yeah, that's where our current base-url stuff falls down I think. The reason I need to provide one here is :v4 isn't want I want (and included in base-url by default). So maybe at this point base-url needs to have a :command-version etc then if we need to override that to test old versions, we would just assoc in a new command-version